docs/shared-content-source/docs/modules/develop/pages/cloudflow-local-sandbox.adoc

```diff
@@ -183,36 +183,62 @@ In a Cloudflow application, Streamlets offer several customization options such
 The Sandbox offers a local implementation of these options that are meaningful in a local environment.
 
 [[local-conf]]
-=== The local configuration file
+=== Using a Local Configuration File
 
-Applications running in the Sandbox can specify custom values for the local environment by making use of a local configuration file in HOCON format. 
-This file is called local.conf by default and is assumed to be available on the classpath, usually under the `application/src/main/resources` folder. 
-A custom local configuration file can be specified in the `build.sbt`, using the `runLocalConfigFile` key. 
-For example, in this `build.sbt` file, we change the local configuration to myruntime.conf in the root dir of the project
+Applications running in the _sandbox_ can specify custom values for the local environment by making use of a local configuration file in HOCON format.
 
-[source,conf]
+This file is usually located in the `src/main/resources` folder of your application project.
+The full relative path to this local configuration file must be specified in the `build.sbt`, using the `runLocalConfigFile` key, like we show in the following snippet: 
+
+.`build.sbt`
+[source, scala]
+----
+include::{cloudflow-examples-version}@docsnippets:ROOT:example$sensor-data-scala/build.sbt[tag=local-conf]
+...
+----
+
+==== Providing Configuration Values
+
+With the local configuration file, you can provide values to streamlet configuration parameters and point volume mounts to locally available folders.
+
+The configuration provided in this file follows the same principles as the general streamlet configuration system in Cloudflow: It uses HOCON syntax to define configuration entries, and it's hierarchically structured by streamlet under the `cloudflow.streamlets.<streamlet-name>` key.
+
+For example, in the following configuration file, we give a local path to a `volume-mount` named `file-ingress` and provide a local value for a configuration parameter called `valid-logger`.
+
+.Example of a `local.conf` file
+[source, HOCON]
+----
+include::{cloudflow-examples-version}@docsnippets:ROOT:example$sensor-data-scala/src/main/resources/local.conf[]
 ----
-runLocalConfigFile := Some("./myruntime.conf"),
-The contents of this file are organized by streamlet name, using the streamlet name that you specified in the blueprint.
 
-file-ingress {
-  //config-key = value  -- values for the file-ingress streamlet
-}
+Note that because this file is in HOCON format, dot-notation is also supported.
 
-rotor-avg-logger {
-  //config-key = value  -- values for the rotor-avg-logger streamlet
-}
+[source, HOCON]
 ----
+cloudflow.streamlets.<streamlet-name>.<config-facet>.<config-key>=value
+----
+
+=== Supported Configuration Facets
 
-Note that because this file is in HOCON format, dot-notation is also supported:
+In the sandbox runtime, only two configuration facets of streamlets are currently supported:
+
+- configuration parameters, under the `config-parameters` facet.
```

Comment:
facet -> section?

Reply:
yeah. I've been looking for a good term for that.
wdyt about _scope_?

Reply:
hocon docs (once!) refers to config sections. I think config section is fine for now.

Reply:
scope is good too.